socialwarehouse/orchestration/asset_factories.py

```
@@ -178,3 +256,43 @@ def _resolve_django_model(app_label: str, model_name: str):
+        MaterializationRecord.objects.create(
+            asset_key=asset_key,
+            source_tier=source_tier,
+            target_table=target_table,
+            source_row_count=source_row_count,
+            target_row_count=target_row_count,
+            is_parity=(source_row_count == target_row_count),
+            duration_seconds=duration_seconds,
+            dagster_run_id=dagster_run_id,
+        )
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)